backend/clubs/views.py

```diff
@@ -4911,6 +4911,19 @@ def complete_checkout(self, request, *args, **kwargs):
             Cart.objects.prefetch_related("tickets"), owner=self.request.user
         )
 
+        # Guard against holds expiring before the capture context
+        tickets = cart.tickets.select_for_update(skip_locked=True).filter(
+            holder=self.request.user, owner__isnull=True
+        )
```

Comment:
Not sure if the `select_for_update` is needed here. I'm emulating the check we do in `initiate_checkout` (which has the lock). Is obtaining a lock necessary in both cases?

Reply:
nope: we don't actually update so we're good :)